runtime/v1/shim/service.go

```diff
@@ -159,6 +159,10 @@ func (s *Service) Create(ctx context.Context, r *shimapi.CreateTaskRequest) (_ *
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
+	if len(mounts) == 0 {
+		rootfs = ""
```

Comment:
`rootfs` should be initialized as an empty string and set to `filepath.Join(r.Bundle, "rootfs")` only when `len(mounts) > 0`?